Traceback (most recent call last):
  File "main.py", line 163, in <module>
    main()
  File "main.py", line 73, in main
    wandb.log({'Model Name': model_name,'DateTime': dt_string})
NameError: name 'model_name' is not defined